### PR TITLE
Add compat data for XSLTProcessor

### PR DESCRIPTION
**Pull request by maboa**
https://developer.mozilla.org/en-US/docs/Web/API/XSLTProcessor

### CODE REVIEW

File: api/XSLTProcessor.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
+{
+  "api": {
+    "XSLTProcessor": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/XSLTProcessor",
+        "support": {
+          "webview_android": {
+            "version_added": null
```
Comment (jpmedley) [inline api/XSLTProcessor.json]: This should be `true`.

File: api/XSLTProcessor.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+          },
+          "chrome": {
+            "version_added": true
+          },
+          "chrome_android": {
+            "version_added": null
```
Comment (jpmedley) [inline api/XSLTProcessor.json]: `true`

File: api/XSLTProcessor.json [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+          },
+          "ie": {
+            "version_added": false
+          },
+          "opera": {
+            "version_added": true
+          },
+          "opera_android": {
+            "version_added": null
```
Comment (jpmedley) [inline api/XSLTProcessor.json]: `true`

File: api/XSLTProcessor.json (hunk 1 above)
Comment (maboa) [reply api/XSLTProcessor.json]: 👍

File: api/XSLTProcessor.json (hunk 2 above)
Comment (maboa) [reply api/XSLTProcessor.json]: 👍

File: api/XSLTProcessor.json (hunk 3 above)
Comment (maboa) [reply api/XSLTProcessor.json]: 👍

**Comment by maboa** [comment]
updated :)

**Pull request merged**